### ISSUE TITLE
DEXSeq HTML report generation failure

### ISSUE THREAD

**Issue by azathoth99** [galaxyproject/galaxy#5352]
The DEXSeq tool on usegalaxy correctly generates results file. However, the HTML report does not include any of the graphical data representations. These were previously available, and now do not load. This appears to be a new problem, as I previously had no issue in using the report.
What I would expect to see:
![image](https://user-images.githubusercontent.com/28160677/35168355-a6147a72-fd26-11e7-8528-52d5b480cc3f.png)
What I see:
![image](https://user-images.githubusercontent.com/28160677/35168374-b5d06ae8-fd26-11e7-97e7-e04ff26fde54.png)

**Comment by dannon** [comment]
@azathoth99 Can you refresh the page and try again for me?

**Comment by azathoth99** [comment]
Yes, the problem remains. The particularly confusing part is that all of the reports have been affected, including the ones that were generated several months ago.

**Comment by dannon** [comment]
@azathoth99 Right, the output reports are fine, it's just changes to the Galaxy display code that have interfered here.   I toggled a setting that would hopefully fix it for you, but it seems there's more for me to do here.  I'll follow up on this and we should be able to get it working again.

**Comment by azathoth99** [comment]
OK, thanks. I'd also note that the problem persists even if I download the entire file and try to use it outside of Galaxy. Not sure if that helps.
